src/core/orderbook.js

```diff
@@ -1300,18 +1296,18 @@ OrderBook.prototype.toJSON =
 OrderBook.prototype.to_json = function() {
   const json = {
     taker_gets: {
-      currency: this._currencyGets.to_hex()
+      currency: this._currencyGets
```

Comment:
I don't see any problem with it; I'm not sure why it was hex in the first place.